compiler/AST/iterator.cpp

```diff
@@ -448,6 +494,7 @@ buildZip2(IteratorInfo* ii, Vec<BaseAST*>& asts, BlockStmt* singleLoop) {
   BlockStmt* zip2body = new BlockStmt();
 
   // Copy non-arg def expressions from the original iterator
+  // See note above.
```

Comment:
Outside of the patch, it won't be obvious what note this is referring to. I think updating the comment to "See note in buildZip1()" is fine.